fuzz/fuzzers/fragment.rs

```diff
@@ -22,13 +22,12 @@ fuzz_target!(|data: &[u8]| {
     };
 
     let frg = fragmenter::MessageFragmenter::new(Some(32)).unwrap();
-    let mut out = VecDeque::new();
-    frg.fragment(
-        message::PlainMessage::from(msg),
-        &mut out,
-    );
-
-    for msg in out {
-        message::Message::try_from(msg).ok();
+    for msg in frg.fragment_message(&message::PlainMessage::from(msg)) {
```

Comment:
Style nit: if this was new code, I think we'd outright import `PlainMessage` and `Message` at the top of the module. Since you're rewriting anyway, maybe do that now?